tests/test_event_handlers.py

```diff
@@ -13,15 +14,43 @@
 from django.test import TestCase
 from mock import patch
 
-from djstripe.models import Event, Charge, Transfer, Account, Plan, Customer, InvoiceItem, Invoice, Card, Subscription
-from tests import (FAKE_CUSTOMER, FAKE_CUSTOMER_II, FAKE_EVENT_CHARGE_SUCCEEDED, FAKE_EVENT_TRANSFER_CREATED,
-                   FAKE_EVENT_PLAN_CREATED, FAKE_CHARGE, FAKE_CHARGE_II, FAKE_INVOICE_II, FAKE_EVENT_INVOICEITEM_CREATED,
-                   FAKE_EVENT_INVOICE_CREATED, FAKE_EVENT_CUSTOMER_CREATED, FAKE_EVENT_CUSTOMER_SOURCE_CREATED,
-                   FAKE_EVENT_CUSTOMER_SUBSCRIPTION_CREATED, FAKE_PLAN, FAKE_SUBSCRIPTION, FAKE_SUBSCRIPTION_III)
 from djstripe.exceptions import CustomerDoesNotExistLocallyException
+from djstripe.models import (
+    Event, Charge, Transfer, Account, Plan, Customer,
+    InvoiceItem, Invoice, Card, Subscription)
+from tests import (
+    FAKE_CARD, FAKE_CHARGE, FAKE_CHARGE_II, FAKE_CUSTOMER, FAKE_CUSTOMER_II,
+    FAKE_EVENT_CHARGE_SUCCEEDED, FAKE_EVENT_CUSTOMER_CREATED,
+    FAKE_EVENT_CUSTOMER_DELETED, FAKE_EVENT_CUSTOMER_SOURCE_CREATED,
+    FAKE_EVENT_CUSTOMER_SOURCE_DELETED,
+    FAKE_EVENT_CUSTOMER_SOURCE_DELETED_DUPE,
+    FAKE_EVENT_CUSTOMER_SUBSCRIPTION_CREATED,
+    FAKE_EVENT_CUSTOMER_SUBSCRIPTION_DELETED, FAKE_EVENT_INVOICE_CREATED,
+    FAKE_EVENT_INVOICE_DELETED, FAKE_EVENT_INVOICEITEM_CREATED,
+    FAKE_EVENT_INVOICEITEM_DELETED, FAKE_EVENT_PLAN_CREATED,
+    FAKE_EVENT_PLAN_DELETED, FAKE_EVENT_TRANSFER_CREATED,
+    FAKE_EVENT_TRANSFER_DELETED, FAKE_INVOICE, FAKE_INVOICE_II,
+    FAKE_INVOICEITEM, FAKE_PLAN, FAKE_SUBSCRIPTION, FAKE_SUBSCRIPTION_III,
+    FAKE_TRANSFER)
```

Comment:
I'll align this as closely as I can, but I added some, that's why the formatting changed.